src/basicElements/serializers/number/integers.ts

```diff
@@ -67,3 +78,44 @@ export function numberToUnsigned(
   mustBeValidUnsigned(sizeInBits, int)
   return int
 }
+
+function setBigUint128(
+  view: DataView,
+  byteOffset: number,
```

Comment:
I'm not sure we need this parameter. It's needed for view.XXX(...), but as we are passing a constant, it shall be easily removed.